battleship.py

```diff
@@ -20,7 +20,7 @@ def print_board():
 
 
 def check_vic(tally):
-    if tally == 17:
+    if tally >= 17:
```

Comment:
Good work but next time make this a separate commit from the other function you changed.